src/nvme/mi.c

```diff
@@ -529,6 +529,30 @@ static void nvme_mi_admin_init_resp(struct nvme_mi_resp *resp,
 	resp->hdr_len = sizeof(*hdr);
 }
 
+static void nvme_mi_control_init_req(struct nvme_mi_req *req,
+				   struct nvme_mi_control_req *control_req, __u8 opcode, __u16 cpsp)
```

Comment:
Super minor, but looks like the wrapping has gone wrong here (and in following prototypes). Only need to address if you're re-rolling for other reasons though!

Reply:
do you means the format, if yes, done~

Reply:
Yep, that's what I was referring to.